templates/accounts.go

```diff
@@ -35,13 +36,13 @@ type Contract struct {
 	Source string
 }
 
-// SourceBytes returns the UTF-8 encoded source code (Source) of the contract.
-func (c Contract) SourceBytes() []byte {
-	return []byte(c.Source)
+// SourceHex returns the UTF-8 encoded source code (Source) of the contract as a hex string.
+func (c Contract) SourceHex() string {
+	return hex.EncodeToString([]byte(c.Source))
 }
 
 const createAccountTemplate = `
-transaction(publicKeys: [[UInt8]], contracts: {String: [UInt8]}) {
+transaction(publicKeys: [[UInt8]], contracts: {String: String}) {
```

Comment:
Are the `publicKeys: [[UInt8]]` also increased 30 bytes per byte?

Reply:
Good catch!